packages/grid/_modules_/grid/hooks/features/preferencesPanel/useGridPreferencesPanel.ts

```diff
@@ -59,6 +71,48 @@ export const useGridPreferencesPanel = (
     'ColumnMenuApi',
   );
 
+  /**
+   * PRE-PROCESSING
+   */
+  const stateExportPreProcessing = React.useCallback<
```

Comment:
To be fair, the preference panel initial state is here because we are using it a lot in tests and it does not hurt for users.
But it's not a very interesting thing to export / restore I agree.